### ISSUE TITLE
build error - make all - openAvnu

### ISSUE THREAD

**Issue by namdala** [Avnu/OpenAvnu#931]
Hello,

I got error using @make clean all
could you help me?

error message : 
Consolidate compiler generated dependencies of target avbTl
[ 96%] Built target avbTl
Consolidate compiler generated dependencies of target openavb_avdecc
[ 98%] Linking C executable openavb_avdecc
/usr/bin/ld: ../../../libavbTl.a(openavb_grandmaster_osal.c.o):(.bss+0x0): multiple definition of `gPtpTD'; ../../../libavbTl.a(openavb_time_osal.c.o):(.bss+0x0): first defined here
collect2: error: ld returned 1 exit status
make[4]: *** [platform/Linux/avb_avdecc/CMakeFiles/openavb_avdecc.dir/build.make:99: platform/Linux/avb_avdecc/openavb_avdecc] Error 1
make[3]: *** [CMakeFiles/Makefile2:306: platform/Linux/avb_avdecc/CMakeFiles/openavb_avdecc.dir/all] Error 2
make[2]: *** [Makefile:136: all] Error 2
make[1]: *** [avtp_avdecc.mk:7: all] Error 2
make: *** [Makefile:138: avtp_avdecc] Error

**Comment by namdala** [comment]
I resolved to modify the code and successfully resolved the issue.
"I didn't encounter any issues when building with the same command in Windows' WSL, but I faced problems when building in Ubuntu 20.04."
I successfully resolved the issue by replacing all occurrences of the variable gPtpTD with gPtpTD_1 in the openavb_grandmaster_osal.c file.
I'm not sure if this solution is correct, but I was able to successfully build it. I hope this helps anyone who encounters a similar issue.

**Comment by namdala** [comment]
I didn't encounter any issues when building with the same command in Windows' WSL, but I faced problems when building in Ubuntu 20.04.
I successfully resolved the issue by replacing all occurrences of the variable gPtpTD with gPtpTD_1 in the openavb_grandmaster_osal.c file.
I'm not sure if this solution is correct, but I was able to successfully build it. I hope this helps anyone who encounters a similar issue.
